myapp/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class SessionsController < ApplicationController
+  before_action :current_user
+  before_action :require_log_in, only: [:destroy, :session_params]
+
+  def new; end
+
+  def create
+    @user = User.find_by(email: session_params[:email])
+    unless @user.present?
```

Comment:
こういう時は、↓のほうがわかりやすいです！
```suggestion
    if @user.blank?
```

Reply:
上記ご指摘の通りです！反映させて頂きました。